addons/FlashCards/src/presenter.js

```diff
@@ -0,0 +1,468 @@
+function AddonFlashcards_create(){

```

Comment:
czy to nie powinno być czasem AddonFlashCards_create? Dopiero po takiej zmianie addon zaczął mi działać poprawnie.